src/plugin.rs

```diff
@@ -119,3 +230,85 @@ impl FmodPlugin {
         Ok(())
     }
 }
+
+fn attributes3d(pos: Vec3, vel: Vec3, fwd: Vec3, up: Vec3) -> Attributes3d {
+    Attributes3d {
+        position: Vector {
+            x: pos.x,
+            y: pos.y,
+            z: -pos.z,
+        },
+        velocity: Vector {
+            x: vel.x,
+            y: vel.y,
+            z: -vel.z,
+        },
+        forward: Vector {
+            x: fwd.x,
+            y: fwd.y,
+            z: -fwd.z,
+        },
+        up: Vector {
+            x: up.x,
+            y: up.y,
+            z: -up.z,
+        },
+    }
+}
+
+impl AudioSinkPlayback for FmodAudioSourcePlayer {
+    fn volume(&self) -> f32 {
+        let (volume, _final_volume) = self.fmod_event.pointer.get_volume().unwrap();
+        volume
+    }
+
+    fn set_volume(&self, volume: f32) {
+        self.fmod_event.pointer.set_volume(volume).unwrap();
+    }
+
+    fn speed(&self) -> f32 {
+        todo!()
+    }
+
+    fn set_speed(&self, speed: f32) {
+        todo!()
+    }
```

Comment:
Having `// TODO`'s is ok, but we should keep the main branch clean of unimplemented API.

Reply:
Implemented the closest thing I could find (increasing the pitch speeds the audio up...)